utils/rt-osgi/src/test/java/org/apache/brooklyn/util/osgi/OsgisTest.java

```diff
@@ -30,12 +28,12 @@
 
     @Test
     public void testParseOsgiIdentifier() throws Exception {
-        assertEquals(Osgis.parseOsgiIdentifier("a.b").get(), new VersionedName("a.b", null));
-        assertEquals(Osgis.parseOsgiIdentifier("a.b:0.1.2").get(), new VersionedName("a.b", Version.parseVersion("0.1.2")));
-        assertEquals(Osgis.parseOsgiIdentifier("a.b:0.0.0.SNAPSHOT").get(), new VersionedName("a.b", Version.parseVersion("0.0.0.SNAPSHOT")));
-        assertFalse(Osgis.parseOsgiIdentifier("a.b:0.notanumber.2").isPresent()); // invalid version
-        assertFalse(Osgis.parseOsgiIdentifier("a.b:0.1.2:3.4.5").isPresent());    // too many colons
-        assertFalse(Osgis.parseOsgiIdentifier("a.b:0.0.0_SNAPSHOT").isPresent()); // invalid version
-        assertFalse(Osgis.parseOsgiIdentifier("").isPresent());
+        assertEquals(OsgiUtils.parseOsgiIdentifier("a.b").get(), new VersionedName("a.b", null));
```

Comment:
The rt-osgi project is a temporary repository for code that is needed both by brooklyn-core and rt-felix. `OsgiUtils` is the class where code was moved from `Osgis`; the name change is supposed to make it easier to distinguish which code needs which project.

The plan is to move it back to core when we decide to drop rt-felix, but... I guess we'll see when we get there.